test/release/examples/primers/modules.chpl

```diff
@@ -0,0 +1,386 @@
+      } // Will output modToUse.foo (which is '12')
+    }
+
+
+    {
+      /* Equivalently, a scope may contain multiple use statements */
+      use modToUse;
+      use AnotherModule, ThirdModule;
+
+      writeln(a && foo > 15);
+      // outputs false (because AnotherModule.a is 'false' and modToUse.foo is
+      // '12')
+    }
+
+
+    {
+      /* In either case, the modules used in this way are considered in concert
+         (after symbols defined at this scope but before symbols defined outside
+         of it) - the ordering within a use statement or across multiple use
+         statements does not affect the precedence of symbols that share a name.
+         This means that if two modules each define a symbol with the same name,
+         and both modules are used at the same scope, attempts to access a
+         symbol by that name will result in a naming conflict:
+      */
+      use modToUse, Conflict;
+
+      writeln(foo); // Outputs modToUse.foo ('12')
+      /* The following line would fail because both modToUse and Conflict define
+         a symbol named bar:
+      */
+      // writeln(bar);
+      writeln(other); // Outputs Conflict.other ('5.0 + 3.0i')
+    }
+
+    {
+      /* When the symbol being accessed is the name of a function, the rules
+         become more complex.  If the two function definitions are overloads (or
+         define different arguments), then the best match will be found, no
+         matter where the function is defined relative to the other function
+         definitions.
+
+         More details on when overloading applies, when functions may shadow
+         other functions, etc. can be found in the relevant section of the
+         language specification.  They will not be covered further in this
+         primer.
+      */
+
+      use modToUse, DifferentArguments;
+
+      writeln(baz(2, 3));
+      // Accesses the function modToUse.baz using the two arguments.  Should
+      // output 2 * (2 + 3) or '10'
+      writeln(baz(3));
+      // Access the function DifferentArguments.baz using the single argument.
+      // Should output 3 - 2, or '1'
+    }
+
+    writeln();
+    writeln("Limiting a use");
+
+
+    {
+      /* To get around such conflicts, there are multiple strategies.  If only a
+         small number of symbols are desired from a particular module, you can
+         specify the symbols to bring in via an 'only' list.
+
+         Here, because of the 'only' clause in the 'use' of Conflict, Conflict's
+         'bar' is not directly accessible here.
+      */
+      use modToUse;
+      use Conflict only other, another;
+
+      writeln(foo); // Outputs modToUse.foo ('12')
+      writeln(bar); // Outputs modToUse.bar ('2')
+      writeln(other); // Outputs Conflict.other ('5.0 + 3.0i')
+    }
+
+
+    {
+      /* If every symbol other than the one which causes a conflict is desired,
+         you can specify the symbols to exclude via an 'except' list.
```

Comment:
This is a bit of an overstatement: you can also do this even if only some of the symbols other than the one that causes the conflict is desired.  Rather than stating in terms of desires, how about just stating the facts.  "Using an 'except' list will cause every symbol other than the ones listed to be available."